src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/ShowPopup/readme.metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+    "category": "Layers",
+    "description": "Show predefined popups from a web map.",
+    "formal_name": "ShowPopup",
+    "ignore": false,
+    "images": [
+        "ShowPopup.jpg"
+    ],
+    "keywords": [
+        "feature",
+        "feature layer",
+        "popup",
+        "web map"
+    ],
+    "nuget_packages": {
+        "Esri.ArcGISRuntime": "100.9.0",
+        "Esri.ArcGISRuntime.WPF": "100.9.0"
```

Comment:
Toolkit package should be added to these (this will prevent sample/doc build issues)